presets/lara/button/index.js

```diff
@@ -14,7 +14,7 @@ export default {
                 'text-xl py-3 px-4': props.size === 'large'
             },
             {
-                'w-12 p-0 py-3': props.label == null && props.icon !== null
+                'w-12 p-0 py-3': props.label == null && props.icon !== null && props.size !== 'small'
```

Comment:
Is there any reason to set `py-3` in this rule? I just encountered this issue an my customization was the simply remove `py-3` from the iconless rule and let the size based y-padding stand.

```suggestion
                'w-12 px-0': props.label == null && props.icon !== null
```